src/index.html

```diff
@@ -7,8 +7,72 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;500&display=swap" rel="stylesheet" 
+    crossorigin="true">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="conteiner" id="home">
+      <li class="nav__img">
+        <a  href="#home">
+          <img
+            class="img"
+            src="images/logo.png"
+            alt="Logo MOYO"
+          >
+        </a>
+      </li>
+      <nav class="navigation">
+        <ul class="navigaion__list">
+          <li class="nav__item">
+            <a class="nav__link is-active"
+               href="."
+            >Apple</a>
```

Comment:
according to previous mentor's remarks. Apply code styling recommendation all over the page. Read [here](https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md)
```suggestion
            <a 
              class="nav__link is-active"
              href="."
            >
              Apple
            </a>
```